packages/models/src/delegation/delegation.ts

```diff
@@ -0,0 +1,40 @@
+import z from "zod";
+import { DelegationId, EServiceId, TenantId } from "../brandedIds.js";
+
+export const delegationKind = {
+  LeadOrganization: "LeadOrganization",
+};
+export const DelegationKind = z.enum([
+  Object.values(delegationKind)[0],
+  ...Object.values(delegationKind).slice(1),
+]);
+export type DelegationKind = z.infer<typeof DelegationKind>;
+
+export const delegationState = {
+  Submitted: "Submitted",
+  Approved: "Approved",
+  Rejected: "Rejected",
+  Revoked: "Revoked",
+};
```

Comment:
```suggestion
} as const;
```
It is needed in order to make typescript assert the values of this object as literal strings